package.json

```diff
@@ -1,42 +1,46 @@
 {
-  "name": "got-sentiment",
-  "version": "0.0.1",
-  "description": "GoT Twitter Sentiment Analysis",
-  "main": "app.js",
-  "scripts": {
-    "test": "grunt test",
-    "start": "node app.js",
-    "hookup": "touch .git/hooks/pre-commit && echo \"npm test\" > .git/hooks/pre-commit && chmod +x .git/hooks/pre-commit"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/Rostlab/JS16_ProjectD_Group4.git"
-  },
-  "keywords": [
-    "got",
-    "sentiment",
-    "twitter",
-    "game",
-    "of",
-    "thrones",
-    "analysis"
-  ],
-  "author": "Santanu Mohanta, Marcus Novotny, Julien Schmidt",
-  "license": "GPL-3.0",
-  "bugs": {
-    "url": "https://github.com/Rostlab/JS16_ProjectD_Group4/issues"
-  },
-  "homepage": "https://github.com/Rostlab/JS16_ProjectD_Group4#readme",
-  "dependencies": {
-    "Sentimental": "^1.0.1",
-    "express": "^4.13.4",
-    "express-handlebars": "^3.0.0",
-    "mongoose": "^4.4.6",
-    "slug": "^0.9.1",
-    "twitter": "^1.2.5"
-  },
-  "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-contrib-jshint": "^1.0.0"
-  }
+    "name": "got-sentiment",
+    "version": "0.0.1",
+    "description": "GoT Twitter Sentiment Analysis",
+    "main": "app.js",
+    "scripts": {
+        "start": "node app.js"
```

Comment:
@julienschmidt An actual, real spacebar or is it a plugin for something? :smile: